lib/ansible/modules/system/firewalld.py

```diff
@@ -197,21 +215,19 @@ class FirewallTransaction(object):
     global module
 
     def __init__(self, fw, action_args=(), zone=None, desired_state=None,
-                 permanent=False, immediate=False, fw_offline=False,
-                 enabled_values=None, disabled_values=None):
-        # type: (firewall.client, tuple, str, bool, bool, bool)
```

Comment:
Why is the definition of `__init__` being changed? Seems out of scope for this change.